### DISCUSSION TITLE
Cannot download latest AIR 33.1.1.779 SDK

### DISCUSSION THREAD

**Discussion by htmiel** [airsdk/Adobe-Runtime-Support#1750] [General]
When attempting to download the latest AIR SDK from https://airsdk.harman.com/download/33.1.1.779, the browser gets redirected to the home page https://airsdk.harman.com/

**Comment by ajwfrost** [comment]
Hi
Thanks for the notification - there was a move across to a new system at the weekend and it seems this folder hadn't synced properly, it's currently re-uploading so should be available again in a few hours.
thanks
